src/components/Home/Home.css

```diff
@@ -0,0 +1,22 @@
+.row {
+  display: flex;
+  justify-content: space-evenly;
+  margin-top: 2rem;
+}
+.MuiCardMedia-img {
+  object-fit: fill !important;
```

Comment:
Is this important required? Why is it needed?

Reply:
needed for image to fit in div, now it's not in use, removed it.